core-bundle/src/Doctrine/ORM/FailTolerantProxyCacheWarmer.php

```diff
@@ -32,9 +32,9 @@ public function isOptional(): bool
      */
     public function warmUp(string $cacheDir): array
     {
-        // If there are no DB credentials yet (install tool) and the
-        // server_version was not configured, we have to skip the ORM warmup to
-        // prevent a DBAL exception during the automatic version detection
+        // If there are no DB credentials yet and the server_version was not
+        // configured, we have to skip the ORM warmup to prevent a DBAL
+        // exception during the automatic version detection
```

Comment:
Can we not remove the try/catch block entirely now?

Reply:
The `FailTolerantProxyCacheWarmer` is also needed in any other (non-Contao) route that needs to work without a working database connection. Now that we have this working wolution, we should IMHO keep it.

Reply:
@ausi WDYT?

Reply:
Ok. 😊